pkgdb/tests/manifest.cc

```diff
@@ -433,12 +468,17 @@ main()
   RUN_TEST( GlobalManifestGA_getRegistryRaw0 );
   RUN_TEST( EnvironmentManifestGA_getRegistryRaw0 );
 
+  RUN_TEST( hookAllowsAtMostOneActivationHook );
+  RUN_TEST( parseManifestRawWithOnActivateScript );
+
   return exitCode;
 }
 
 
-/* -------------------------------------------------------------------------- *
+/* --------------------------------------------------------------------------
+ * *
  *
  *
  *
- * ========================================================================== */
+ * ==========================================================================
+ */
```

Comment:
?

these comment blocks are ... special

Reply:
The autoformatter makes these tweaks when you're mid-edit and then never puts them back 🙄